app/Http/Controllers/ProjectsController.php

```
@@ -158,8 +172,38 @@ public function store(StoreProjectRequest $request)
+
+        return view('projects.edit')->withProject($this->findByExternalId($external_id));
+    }
+
+    public function update($external_id, UpdateProjectRequest $request)
+    {
+        $project = $this->findByExternalId($external_id);
+        $this->projectService->update($project, $request->validated());
+        session()->flash('flash_message', __('Project successfully updated'));
+
+        return redirect()->route('projects.show', $project->external_id);
     }
```

PHPMD (2.15.0)

[error] The parameter $external_id is not named in camelCase. (undefined)

(CamelCaseParameterName)